Makefile

```
@@ -97,6 +99,14 @@ monitoring/uninstall:
 	- kubectl delete -f deploy/monitor/prometheus-rule.yaml
 	- kubectl delete -f deploy/monitor/grafana-dashboard.yaml
 
+##############################
+# CI                         #
+##############################
+
+.PHONY: code/build/linux
```

```suggestion
.PHONY: binary/build/linux
```

Not sure what task name feels right here, but `code` seems odd for some reason

---

Ignore above, I see if fits the task naming convention we're adopting